.github/workflows/pypi-publish.yml

```
@@ -43,10 +43,10 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: 3.9
       - uses: ./.github/workflows/build_linux_wheel
         with:
-          python-minor-version: 8
+          python-minor-version: 9
```

actionlint (1.7.4)

input "python-minor-version" is not defined in action "build-linux-wheel" defined at "./.github/workflows/build_linux_wheel". available inputs are "args", "arm-build", "manylinux"

(action)